taskcluster/requirements.in

```diff
@@ -0,0 +1,5 @@
+# For instructions on managing dependencies, see:
+# https://taskcluster-taskgraph.readthedocs.io/en/latest/howto/bootstrap-taskgraph.html
+
+taskcluster-taskgraph-hneiva@https://test-files.pythonhosted.org/packages/21/e2/9efb5cc8b6b5ab4cc5d46a5338c4c02fffebe8d06de6de789b8017c0c35d/taskcluster_taskgraph_hneiva-1.3.0rc1-py3-none-any.whl
```

Comment:
Don't forget to change this :)

You can also point this at a `taskgraph-try` push for future reference:
https://taskcluster-taskgraph.readthedocs.io/en/latest/howto/bootstrap-taskgraph.html#testing-pre-release-versions-of-taskgraph

Reply:
Unfortunately, you cannot.
When installing the requirements file, it's setup to require hashes, and pip cannot verify hashes when pointing to a repository.
`ERROR: Can't verify hashes for these requirements because we don't have a way to hash version control repositories:`

[1] Where we declare `--require-hashes`: https://hg.mozilla.org/ci/taskgraph/file/tip/src/taskgraph/run-task/run-task#l809

I could have modified that in my revision, but then I felt like there was too many hacks on top of hacks.

Reply:
Hmm, are you using an older version of pip tools or something? I had tested this out before and it wfm. Will try again in a bit